airsonic-main/src/main/java/org/airsonic/player/service/metadata/FFmpegParser.java

```diff
@@ -144,7 +144,7 @@ public boolean isApplicable(File file) {
                 return true;
             }
         }
-        return false;
+        return "ape".equals(format);
```

Comment:
I'm not super-duper comfortable with this kind of hardcoding. Could you please make it a bit more generic, so that we can do the same for more file formats?

Reply:
hmm...I don't quite understand. This is just handle the format in the similar way of [JaudiotaggerParser.java](https://github.com/airsonic/airsonic/blob/ac0a722a105612b652df311600c94f2ea93cc8b2/airsonic-main/src/main/java/org/airsonic/player/service/metadata/JaudiotaggerParser.java#L298)